homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -270,3 +273,42 @@ async def async_select_source(self, source):
     async def async_select_sound_mode(self, sound_mode):
         """Select EQ Preset."""
         await self.fs_device.set_eq_preset(self.__sound_modes_by_label[sound_mode])
+
+    async def async_browse_media(self, media_content_type=None, media_content_id=None):
+        """Browse media library and preset stations."""
+        if media_content_type in (None, "library"):
+            return await browse_top_level(self._attr_source, self.fs_device)
+
+        return await browse_node(self.fs_device, media_content_type, media_content_id)
+
+    async def async_play_media(self, media_type, media_id, **kwargs):
+        """Play selected media or channel."""
+        supported_media_types = [MEDIA_TYPE_CHANNEL, MEDIA_TYPE_PRESET]
```

Comment:
This can be moved to your separate const.py

Reply:
I moved it to `const.py`, but feel like this makes the code somewhat harder to read. I conscientiously defined that value there directly as it fits the flow of the code, and as the constant is not used anywhere else.